packages/@react-spectrum/actionbar/src/ActionBar.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ActionButton} from '@react-spectrum/button';
+import {ActionGroup} from '@react-spectrum/actiongroup';
+import {classNames, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import CrossLarge from '@spectrum-icons/ui/CrossLarge';
+import {DOMRef} from '@react-types/shared';
+import {filterDOMProps} from '@react-aria/utils';
+import {Provider, useProviderProps} from '@react-spectrum/provider';
+import React from 'react';
+import {SpectrumActionBarProps} from '@react-types/actionbar';
+import styles from './actionbar.css';
+import {Text} from '@react-spectrum/text';
+
+function ActionBar(props: SpectrumActionBarProps, ref: DOMRef<HTMLDivElement>) {
+  // Grabs specific props from the closest Provider (see https://react-spectrum.adobe.com/react-spectrum/Provider.html#property-groups). Remove if your component doesn't support any of the listed props.
+  props = useProviderProps(props);
+
+  const {
+    children,
+    isEmphasized,
+    onAction,
+    onClearSelection,
+    selectedItemCount,
+    variant,
+    ...otherProps
+  } = props;
+
+  // Handles RSP specific style options, UNSAFE_style, and UNSAFE_className props (see https://react-spectrum.adobe.com/react-spectrum/styling.html#style-props)
+  let {styleProps} = useStyleProps(props);
+  let domRef = useDOMRef(ref);
+  let providerProps = {isEmphasized};
+
+  return (
+    <div
+      {...filterDOMProps(props)}
+      {...styleProps}
+      ref={domRef}
+      className={classNames(
+        styles,
+        'spectrum-ActionBar', {
+          'spectrum-ActionBar--primary': variant === 'primary',
```

Comment:
I just thought that, since the wireframe had a light background with dark text and the design had the primary color with white text, there might be variants involved I'm currently unaware of. So I already added the logic for those while testing out stuff :) You should just tell me if you want it to be left there or removed 😄

Reply:
Ah, yep, no variants planned at this time, only emphasized to change to the blue background